src/recommended-alarms/rds-free-storage-space-alarm.ts

```diff
@@ -0,0 +1,53 @@
+import { Duration } from 'aws-cdk-lib';
+import { ComparisonOperator } from 'aws-cdk-lib/aws-cloudwatch';
+import { DatabaseInstance } from 'aws-cdk-lib/aws-rds';
+import { Construct } from 'constructs';
+import { CreateRecommendedAlarmProps, RecommendedAlarm } from './recommended-alarms';
+
+/**
+ * Properties needed to create an RDS free storage space alarm.
+ */
+export interface RdsFreeStorageSpaceAlarmProps extends CreateRecommendedAlarmProps {
+  /**
+   * The database instance to monitor.
+   */
+  database: DatabaseInstance;
+  /**
+   * The minimum amount of free storage space that is allowed before triggering
+   * the alarm.
+   *
+   * @remarks
+   * The threshold value will depend on the currently allocated storage space.
+   * Typically, the threshold value should be 10% of the allocated storage
+   * space.
+   */
+  threshold: number;
```

Comment:
Ideally a default for `threshold` could be determined to simplify usage of this alarm.